build.sbt

```diff
@@ -160,13 +163,14 @@ def module(name: String) = {
 }
 
 lazy val algebirdCore = module("core").settings(
-  test := { }, // All tests reside in algebirdTest
```

Comment:
I put some basic tests of the algebra interop here (they don't use any of the test properties we export, so I think it is the right place).